packages/sitecore-jss/src/data-api.ts

```diff
@@ -1,5 +1,5 @@
-import { LayoutServiceData, PlaceholderData } from './dataModels';
-import { HttpJsonFetcher, HttpResponse } from './httpClientInterface';
+import { LayoutServiceData, PlaceholderData } from './data-models';
+import { HttpDataFetcher, HttpResponse } from './data-fetcher';
```

Comment:
What do you think is the issue with moving it to "data-fetcher"? The reason I made this change is because
1. The class name is HttpDataFetcher, and it's better for filename to align with export names
2. The class is responsible for fetching data from an arbitrary endpoint; it's not responsible for specifying the endpoint. So "data fetcher" is a more accurate description of what it's doing
3. This change is not a breaking change; it's totally backwards compatible

Reply:
@ambrauer - when I started working on GraphQLDictionaryService, all stuff relating to dictionary was in one file, and I found that the file was so huge that searching for the thing I needed was taking longer than normal. Also, it was more difficult to see what code is shared between the different types of dictionary services, and what is different. This is why I chose to break them up.

Reply:
@anastasiya29 Got it, I'm cool with that then. In that case, have a couple other questions:
1. Not 100% sold on the "grapqhl." file name prefix. Why not just `graphql-dictionary-service.ts` and `rest-dictionary-service.ts`?
2. Should we apply the same convention for layout service for consistency (i.e. `graphql-layout-service` and `rest-layout-service.ts`)?

Reply:
Sounds good. Will do.